plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseHiveConnectorTest.java

```diff
@@ -1575,6 +1576,20 @@ private void testCreatePartitionedTable(Session session, HiveStorageFormat stora
         assertFalse(getQueryRunner().tableExists(session, "test_partitioned_table"));
     }
 
+    protected Object[][] likeTableParentTableCandidatesData()
+            throws Exception
+    {
+        java.nio.file.Path tempDir = createTempDirectory(null);
```

Comment:
make sure that the test resources you use are being cleaned after the test run (in this case - the temp directory gets deleted)

Reply:
I refactored the code to just add another test in Hive reusing the existing logic, I think it's much simpler and transparent this way.